libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -548,16 +567,21 @@ void AC_AttitudeControl::input_rate_bf_roll_pitch_yaw_cds(float roll_rate_bf_cds
 void AC_AttitudeControl::input_rate_bf_roll_pitch_yaw_2_cds(float roll_rate_bf_cds, float pitch_rate_bf_cds, float yaw_rate_bf_cds)
 {
     // Convert from centidegrees on public interface to radians
-    float roll_rate_rads = radians(roll_rate_bf_cds * 0.01f);
-    float pitch_rate_rads = radians(pitch_rate_bf_cds * 0.01f);
-    float yaw_rate_rads = radians(yaw_rate_bf_cds * 0.01f);
+    float roll_rate_bf_rads = radians(roll_rate_bf_cds * 0.01f);
```

Comment:
nitpick: you could make these const although the original method didn't use const either